### DISCUSSION TITLE
Load Tests  vs Monitoring (can I use HammerDB for monitoring as well)

### DISCUSSION THREAD

**Discussion by Alex-MSP** [TPC-Council/HammerDB#214] [Q&A] [answered]
Hi everbody,

the main purpose of HammerDB is clear but in a fist step: Can I also use it to monitor an existing Oracle DB
and get an idea about load and perfromance without creating Load Tests. 

There is an ERP/GUI running on top of this oracle DB and a bunch of gui users usiing it.

Thanks for all answers
Alex

**Comment by sm-shaw** [comment, marked as answer]
Yes you can. If you look at the documentation here https://www.hammerdb.com/docs/ch07s04.html then there is a guide on how to use the Oracle Metrics feature. This is not restricted to just monitoring the HammerDB workload but can be used visually monitoring any workload on your database. The best approach is to connect to your database and then drag the metrics tab out and then maximise it to its full extent as shown below.  The Oracle Metrics feature uses the Active Session History (ASH) so you can drag the box forwards and backwards in the graph to highlight an area of the window to show the workload at that time. 

Similarly, the transaction counter will also show the TPM for any application and not just the HammerDB workload. You can also connect the agent to observe the user and system CPU load per core. 

![ch7-14](https://user-images.githubusercontent.com/38044085/110332113-d620a980-8017-11eb-8fb2-e3b71c4c395c.PNG)
